src/bijectors/shift.jl

```diff
@@ -1,37 +1,26 @@
 #################
 # Shift & Scale #
 #################
-struct Shift{T, N} <: Bijector{N}
+struct Shift{T} <: Bijector
     a::T
 end
 
-Base.:(==)(b1::Shift{<:Any, N}, b2::Shift{<:Any, N}) where {N} = b1.a == b2.a
+Base.:(==)(b1::Shift, b2::Shift) = b1.a == b2.a
 
-function Shift(a::Union{Real,AbstractArray}; dim::Val{D} = Val(ndims(a))) where D
-    return Shift{typeof(a), D}(a)
-end
-
-# field is a numerical parameter
-function Functors.functor(::Type{<:Shift{<:Any,N}}, x) where N
-    function reconstruct_shift(xs)
-        return Shift{typeof(xs.a),N}(xs.a)
-    end
-    return (a = x.a,), reconstruct_shift
-end
+Functors.@functor Shift
 
-up1(b::Shift{T, N}) where {T, N} = Shift{T, N + 1}(b.a)
+Base.show(io::IO, b::Shift) = print(io, "Shift($(b.a))")
```

Comment:
Are such definitions really needed? Isn't that (basically) what you get by default?

Reply:
Without the type parameter, yeah. I'll remove this here though; I think was just playing around with whether or not we should do this and left it in this PR by a mistake.